silkrpc/core/blocks_test.cpp

```diff
@@ -44,38 +45,44 @@ class MockDatabaseReader : public core::rawdb::DatabaseReader {
 };
 
 TEST_CASE("get_block_number", "[silkrpc][core][blocks]") {
+    SILKRPC_LOG_STREAMS(null_stream(), null_stream());
     const silkworm::ByteView kExecutionStage{stages::kExecution};
     MockDatabaseReader db_reader;
     asio::thread_pool pool{1};
 
     SECTION("kEarliestBlockId") {
-        auto result = asio::co_spawn(pool, get_block_number(kEarliestBlockId, db_reader), asio::use_future);
+        static const std::string EARLIEST_BLOCK_ID = kEarliestBlockId;
+        auto result = asio::co_spawn(pool, get_block_number(EARLIEST_BLOCK_ID, db_reader), asio::use_future);
         CHECK(result.get() == kEarliestBlockNumber);
     }
 
     SECTION("kLatestBlockId") {
+        static const std::string LATEST_BLOCK_ID = kLatestBlockId;
         EXPECT_CALL(db_reader, get(db::table::kSyncStageProgress, kExecutionStage)).WillOnce(InvokeWithoutArgs(
             []() -> asio::awaitable<KeyValue> { co_return KeyValue{silkworm::Bytes{}, *silkworm::from_hex("1234567890123456")}; }
         ));
-        auto result = asio::co_spawn(pool, get_block_number(kLatestBlockId, db_reader), asio::use_future);
+        auto result = asio::co_spawn(pool, get_block_number(LATEST_BLOCK_ID, db_reader), asio::use_future);
         CHECK(result.get() == 0x1234567890123456);
     }
 
     SECTION("kPendingBlockId") {
+        static const std::string PENDING_BLOCK_ID = kPendingBlockId;
         EXPECT_CALL(db_reader, get(db::table::kSyncStageProgress, kExecutionStage)).WillOnce(InvokeWithoutArgs(
             []() -> asio::awaitable<KeyValue> { co_return KeyValue{silkworm::Bytes{}, *silkworm::from_hex("1234567890123456")}; }
         ));
-        auto result = asio::co_spawn(pool, get_block_number(kPendingBlockId, db_reader), asio::use_future);
+        auto result = asio::co_spawn(pool, get_block_number(PENDING_BLOCK_ID, db_reader), asio::use_future);
         CHECK(result.get() == 0x1234567890123456);
     }
 
     SECTION("number in hex") {
-        auto result = asio::co_spawn(pool, get_block_number("0x12345", db_reader), asio::use_future);
+        static const std::string BLOCK_ID_HEX = "0x12345";
```

Comment:
why static? and why not directly pass it to the method?

Reply:
Same as above